LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/BasicsStation/ModuleConnection/ModuleConnectionHost.cs

```diff
@@ -149,6 +157,45 @@ private async Task<MethodResponse> ClearCacheAsync()
             return new MethodResponse((int)HttpStatusCode.OK);
         }
 
+        private async Task<MethodResponse> CloseConnectionAsync(MethodRequest methodRequest)
+        {
+            ReceivedLoRaCloudToDeviceMessage c2d = null;
+
+            try
+            {
+                c2d = JsonSerializer.Deserialize<ReceivedLoRaCloudToDeviceMessage>(methodRequest.DataAsJson, new JsonSerializerOptions { PropertyNameCaseInsensitive = true });
+            }
+            catch (Exception ex) when (ex is ArgumentNullException or JsonException)
```

Comment:
Don't think we should be mixing the two cases: `ArgumentNullException` means there's no JSON data and `JsonException` means the JSON syntax or data is invalid.

Reply:
Changed with 0be1f5b3cda6ab3abbfb0bb1b65006befb7139a5